AnwesenheitsSimulation/module.php

```diff
@@ -285,6 +291,7 @@ public function UpdateTargets() {
 			$this->SetTimerInterval("UpdateTargetsTimer", 0);
 		}
 
+		$this->SetTimerInterval("UpdateTargetsTimer", 1000 * (24 * 60 * 60));
```

Comment:
Nicht fest auf 24 Stunden setzen
- Die Ausführung des Skripts kann Zeit in Anspruch nehmen
- Manche Tage sind 23 oder 25 Stunden lang